plugins/acr/package.json

```diff
@@ -13,8 +13,12 @@
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
     "pluginId": "acr",
-    "pluginPackage": "@janus-idp/backstage-plugin-acr"
+    "pluginPackage": "@janus-idp/backstage-plugin-acr",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-acr"
+    ]
```

Comment:
Weird that it has both of these now. Looking at an upstream plugin that only has a frontend plugin associated with it, I see just [pluginPackages](https://github.com/backstage/community-plugins/blob/main/workspaces/grafana/plugins/grafana/package.json#L25:L27)

Reply:
well, I suspect the backstage cli is dumb and doesn't look for BOTH `pluginPackage` and `pluginPackages` when it's adding `pluginPackages`. 

We might not need both.